xbmc/video/dialogs/GUIDialogAudioSubtitleSettings.cpp

```diff
@@ -156,6 +158,16 @@ void CGUIDialogAudioSubtitleSettings::OnSettingChanged(const CSetting *setting)
       g_application.m_pPlayer->SetAudioStream(m_audioStream);    // Set the audio stream to the one selected
     }
   }
+  else if (settingId == SETTING_VIDEO_STREAM)
```

Comment:
They can be different bit rates, but they can also be different camera angles, different special effects, or any number of things. It is a multipurpose feature in mkv's.